tutormfe/patches/caddyfile

```diff
@@ -1,5 +1,5 @@
 {{ MFE_HOST }}{$default_site_port} {
-    respond / 204
+    redir / {% if ENABLE_HTTPS %}https://{% else %}http://{% endif %}{{ LMS_HOST }}{$default_site_port}
```

Comment:
I think you need to remove the `{$default_site_port}` part. Otherwise it will redirect to the port that is used internally.

Reply:
Yup right, I have tried by removing it. It's working fine.